doc/docs/usage/quickstart.md

```diff
@@ -69,20 +69,22 @@ desiredState:
       dependencies: {}
       restartPolicy: ALWAYS
       runtime: podman
-      runtimeConfig: |
+      runtimeConfig: |-
```

Comment:
I think it was copied from the cli output. We do not need it.

Reply:
I understand, but since we have all other examples without the `-`, I would not add it here (or we should change everything else):
```suggestion
      runtimeConfig: |
```